airbyte-lib/tests/integration_tests/fixtures/source-test/source_test/run.py

```diff
@@ -35,6 +35,20 @@
                     },
                 },
             },
+            {
+                "name": "stream3",
+                "supported_sync_modes": ["full_refresh", "incremental"],
+                "source_defined_cursor": False,
+                "json_schema": {
```

Comment:
If this is a test stream that is intended to always be empty, what do you think of us changing the name and adding a description.

```suggestion
            {
                "name": "always-empty-stream",
                "description": "This stream always emits zero records, to test handling of empty datasets."
                "supported_sync_modes": ["full_refresh", "incremental"],
                "source_defined_cursor": False,
                "json_schema": {
```

Reply:
It does break the 'stream1', 'stream2', ... convention, so I guess I could see a case for going either way.

Reply:
yep, makes sense, so we don't accidentally add records for it later. will change. will add comments explaining that too.,